<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModImporter</class>
 <widget class="QDialog" name="ModImporter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mod Importer</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="infoPage">
      <attribute name="label">
       <string>Information</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>If you downloaded a *.zip file, you can try to import it directly</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnImportZip">
            <property name="minimumSize">
             <size>
              <width>170</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Import from *.zip</string>
            </property>
            <property name="icon">
             <iconset theme="document-import"/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>If you have a folder with mod content</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnImportDirectory">
            <property name="minimumSize">
             <size>
              <width>170</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Import from directory</string>
            </property>
            <property name="icon">
             <iconset theme="document-import"/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If you know what to do or automatic importing fails. &lt;a href=&quot;https://github.com/rumangerst/SilVerPLuM/wiki/Modding-tutorial:-Use-the-mod-importer&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#2980b9;&quot;&gt;Here&lt;/span&gt;&lt;/a&gt; you can find a tutorial.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnImportManually">
            <property name="minimumSize">
             <size>
              <width>170</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Import manually</string>
            </property>
            <property name="icon">
             <iconset theme="document-import"/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="modInfoPage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>813</width>
        <height>484</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Mod information</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Identifier</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="IdentifierLineEdit" name="modIdentifier">
         <property name="toolTip">
          <string>The identifier of this modification.</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_7">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="MarkdownTextEditor" name="modDescription">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>200</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_8">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Stardew Valley version</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QComboBox" name="modRequiresStardewValley">
         <property name="toolTip">
          <string>The Stardew Valley version this mod is intended to run.</string>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="modName">
         <property name="toolTip">
          <string>The name of this modification</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Author</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="modAuthor">
         <property name="toolTip">
          <string>The author of this modification.</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>License</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="modLicense">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Select the license used by the modification. The drop-down-box contains pre-made entries you can use.</string>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>Creative Commons CC BY</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Creative Commons CC BY-SA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Creative Commons CC BY-ND</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Creative Commons CC BY-NC</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Creative Commons CC BY-NC-SA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Creative Commons CC BY-NC-ND</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GPLv2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GPLv3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>LGPL</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MIT</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Apache</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>BSD</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Chucklefish Allow-Pack Anyone</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Chucklefish Ask-Pack Anyone</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Chucklefish No-Pack Anyone</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Chucklefish Allow-Pack Ask</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Chucklefish Ask-Pack Ask</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Chucklefish No-Pack Ask</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Chucklefish Allow-Pack Only-Use</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Chucklefish Ask-Pack Only-Use</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Chucklefish No-Pack Only-Use</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Version</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="modVersion">
         <property name="toolTip">
          <string>The version of this mod.</string>
         </property>
         <property name="text">
          <string>1.0.0.0</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_6">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Website</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="modWebsite">
         <property name="toolTip">
          <string>The website URL of this mod.</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="contentItemsPage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>833</width>
        <height>416</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Mod content</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QWidget" name="contentItems" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnAddContent">
        <property name="toolTip">
         <string>Adds another content entry into 'Mod content'.</string>
        </property>
        <property name="text">
         <string>Add content</string>
        </property>
        <property name="icon">
         <iconset theme="list-add">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnClose">
        <property name="text">
         <string>Close</string>
        </property>
        <property name="icon">
         <iconset theme="dialog-close">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnImport">
        <property name="text">
         <string>Import</string>
        </property>
        <property name="icon">
         <iconset theme="dialog-ok">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MarkdownTextEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>markdowntexteditor.h</header>
  </customwidget>
  <customwidget>
   <class>IdentifierLineEdit</class>
   <extends>QLineEdit</extends>
   <header>identifierlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
